test/schematype.test.js

```diff
@@ -211,6 +211,21 @@ describe('schematype', function() {
       it('SchemaType.set, is a function', () => {
         assert.equal(typeof mongoose.SchemaType.set, 'function');
       });
+      it('should allow setting values to a given property gh-13510', async function() {
+        const m = new mongoose.Mongoose();
+        await m.connect('mongodb://127.0.0.1:27017/gh13510');
```

Comment:
Why did you hard code the connection string here? There's nowhere else in the tests, other than a very old sharding test, where we hard code the connection string. Use `mongoose.connect(start.uri)` instead.